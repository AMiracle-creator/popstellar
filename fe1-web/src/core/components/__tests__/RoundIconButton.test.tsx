import { render } from '@testing-library/react-native';
import React from 'react';

import { RoundIconButton } from '../index';

describe('RoundIconButton', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<RoundIconButton name="close" onClick={() => {}} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
