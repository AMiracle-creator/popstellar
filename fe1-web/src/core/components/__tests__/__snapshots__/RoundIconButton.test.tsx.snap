// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundIconButton renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  nativeID="animatedComponent"
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2196F3",
      "borderRadius": 30,
      "display": "flex",
      "height": 65,
      "justifyContent": "center",
      "opacity": 1,
      "shadowColor": "gray",
      "shadowOffset": Object {
        "height": 2,
        "width": 2,
      },
      "width": 65,
    }
  }
>
  <View
    style={
      Object {
        "overflow": "hidden",
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "justifyContent": "center",
          }
        }
      >
        <Text />
      </View>
    </View>
  </View>
</View>
`;
